sdk/trafficmanager/Azure.ResourceManager.TrafficManager/tests/Scenario/ProfileTestBase.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TrafficManager.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.TrafficManager.Tests
+{
+    public abstract class ProfileTestBase : TrafficManagerManagementTestBase
+    {
+        protected internal const string ExpectedKey = "tagKey";
+        protected internal const string ExpectedValue = "tagValue";
+        protected internal const string EndpointName1 = "endpoint1";
+        protected internal const string EndpointName2 = "endpoint2";
+        protected internal const string EndpointTypeName = "externalEndpoints";
+        protected internal const string EndpointType = "Microsoft.Network/trafficManagerProfiles/" + EndpointTypeName;
+
+        protected internal SubscriptionResource _subscription;
+        protected internal ProfileCollection _profileCollection;
+        protected internal ResourceGroupResource _resourceGroup;
+        protected internal string _profileName;
+
+        public ProfileTestBase(bool isAsync, RecordedTestMode mode = RecordedTestMode.Playback) : base(isAsync, mode)
```

Comment:
lets not hard code playback

Reply:
If we want to exclude a test from running in live use the `PlaybackOnlyAttribute`